service-worker/worker/src/test/unit/worker.spec.ts

```diff
@@ -83,6 +83,7 @@ function errored(err, done) {
 function expectOkResponse(value: Response): Response {
   expect(value).not.toBeUndefined();
   expect(value.ok).toBeTruthy();
+  expect(value['redirected']).toBeFalsy();
```

Comment:
I suppose you're intentionally accessing the property with the square brackets syntax?